packages/requisitions/src/utils.ts

```diff
@@ -16,10 +17,21 @@ export const placeholderSupplierRequisition: SupplierRequisition = {
   otherPartyId: '',
   otherPartyName: '',
   status: SupplierRequisitionNodeStatus.Draft,
-  storeId: '',
+  update: () => {
+    throw new Error(
+      'Placeholder callback update has been triggered. This should never happen!'
+    );
+  },
+  updateOtherParty: () => {
+    throw new Error(
+      'Placeholder callback updateOtherParty has been triggered. This should never happen!'
+    );
```

Comment:
Also think I like this but not sure.

The idea being that the draft is created 'properly' after merging the server data in with the draft where the callbacks are created - if this doesn't happen well you shouldn't be trying to update the draft!

Don't usually do things like this ("should not happen or throw") so not sure 🤔

Reply:
could be useful in development phase, perhaps when moving into production a bugsnag notification or other log entry would be more helpful.